Source/Scene/Scene.js

```diff
@@ -287,6 +287,10 @@ define([
         this._primitives = new PrimitiveCollection();
         this._groundPrimitives = new PrimitiveCollection();
 
+        this._logDepthBuffer = undefined;
+        this._logDepthBufferDirty = true;
+        this.logarithmicDepthBuffer = context.fragmentDepth;
```

Comment:
Should this have the same name as the property get/set below?

Reply:
Yes, this calls the setter below.